synapseclient/core/upload/__init__.py

```diff
@@ -0,0 +1,8 @@
+# These are exposed functions and objects from the `synapseclient.core.upload` package.
+# However, these functions and objects are not public APIs for the Synapse Python client.
+# The Synapse Engineering team is free to change their signatures and implementations anytime.
+# Please use them at your own risk.
+
+from .upload_functions import upload_file_handle, upload_synapse_s3
+from .multipart_upload import multipart_upload_file, multipart_upload_string, MIN_PART_SIZE
```

Comment:
kind of weird that modules outside of the package would need to know what MIN_PART_SIZE is. pleas double check what is using it and whether we can refactor that logic back inside this module